src/shared/api/services/ActivityItemDto.ts

```diff
@@ -477,6 +478,11 @@ export interface TextItemDto extends ActivityItemDtoBase {
   config: TextConfiguration;
   responseValues: TextAnswerSettings;
 }
+export interface paragraphTextItemDto extends ActivityItemDtoBase {
+  responseType: 'paragraphText';
+  config: TextConfiguration;
```

Comment:
This is using `TextConfiguration` which includes more options than should be available to `ParagraphItem`. You should create its own `ParagraphConfiguration`